libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -118,6 +115,13 @@ const AP_Param::GroupInfo AC_AutoTune_Multi::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("MIN_D", 3, AC_AutoTune_Multi, min_d,  0.0005f),
 
+    // @Param: GAIN_BK
+    // @DisplayName: AutoTune Gain Backoff (Margin)
+    // @Description: Fraction by which tuned P and D gains are reduced after rate and angle AutoTune steps complete. This provides extra stability margin by reducing gains slightly from the optimal values found during tuning. A value of 0.0 applies no reduction. A value of 0.25 reduces tuned gains by 25%.
+    // @Range: 0.0 0.5
+    // @User: Standard
+    AP_GROUPINFO("GAIN_BK", 4, AC_AutoTune_Multi, gain_backoff,  0.25),
```

Comment:
I'm uneasy about the default being different to the status quo, but I can see the safety argument.

Reply:
What is the "status quo" and what do you think we should use?

Reply:
Status quo is the current backoff, which is less correct?